source/agora/node/Ledger.d

```diff
@@ -94,6 +97,8 @@ public class Ledger
             storage = the block storage
             enroll_man = the enrollmentManager
             pool = the transaction pool
+            onAcceptedBlock = optional delegate to call
```

Comment:
```suggestion
            onAcceptedBlock = delegate to call
```

Reply:
Ah @TrustHenry I didn't realize this was also used in unittests.

Maybe making it optional was actually a good idea. I'm sorry to give you more work, it might be worth reverting the change and making it optional (with the `!is null` check).

Reply:
The code has been reverted. OnAcceptedBlock is set to Optional. with the !is null check
Thanks for your review.